Makefile

```diff
@@ -8,3 +8,13 @@ default:
 
 refresh-piplock-files:
 	cd caikit && pipenv lock
+
+.PHONY: docker-test
+
+docker-test: default
+	podman run -it --rm \
+	-e DTYPE_STR=float32 \
+	--name caikit-tgis-serving-test-$$(git rev-parse --short HEAD) \
+	--volume $$(pwd)/test:/tmp/test:z --volume $$(pwd)/utils:/tmp/utils:z \
+	caikit-tgis-serving:$$(git rev-parse --short HEAD) \
```

Comment:
As far as I see this script requires the image to be already build, does it make sense to build the image as part of this script to have everything in a single command?

Reply:
Yes, build is a dependency of this target https://github.com/opendatahub-io/caikit-tgis-serving/pull/34/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R14